### DISCUSSION TITLE
Firefox's "Do Not Track" feature is ignored

### DISCUSSION THREAD

**Discussion by xuhdev** [plausible/analytics#646] [Anything Else]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report
**Describe the bug**
Set [Firefox's "Do Not Track" feature](https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature) to "always". Visiting a page uses plausible. The visit is still tracked. (Tested on https://plausible.io)

**Expected behavior**
The visit should not be tracked.

**Environment (If applicable):**
 - OS: Linux
 - Browser Firefox
 - Version 84.0.2

**Comment by Vigasaurus** [comment]
That's not exactly how DNT is intended to work; largely, its meaning is to say that users should not be tracked or have any data retained on them that isn't inherently privacy friendly or anonymized fully.

> It also includes exceptions that respect the ordinary functionality of a site, that allow measures for security purposes and prevent fraud, and **allow data analysis techniques that protect the anonymity of the users**.

https://www.eff.org/issues/do-not-track

Plausible does fall under the last point there in most definitions, given the strict protection/lack of retention of user data. 
 https://plausible.io/privacy-focused-web-analytics

There could of course be good reason for Plausible to adhere to a stricter set of DNT rules, but I'd argue that wouldn't be fully intended or required behavior as per DNT standards.

**Comment by xuhdev** [comment]
Thanks! Could you document this somewhere? It isn't obvious that DNT should be ignored by plausible.

**Reply by metmarkosaric** [reply]
Thanks for details @Vigasaurus! 

I'll see if I can add some info to our docs @xuhdev.

**Comment by mirdaki** [comment]
@Vigasaurus Would you consider letting site owners decide whether to get analytics for DNT users or not? It could happen by default, but I at least prefer to not do anything for DNT users. Similar to this [Matomo setting](https://matomo.org/docs/privacy-how-to/#step-4-respect-donottrack-preference) or this [umami setting](https://umami.is/docs/tracker-config)?

**Reply by Vigasaurus** [reply]
I just contribute things 🤷‍♂️; that'd be a decision for @ukutaht & @metmarkosaric 

In my opinion it couldn't hurt, though

**Reply by mirdaki** [reply]
Ah, should of checked that. Thanks for the clarification!

**Reply by metmarkosaric** [reply]
thanks @mirdaki! there's no plans to do this. DNT is not relevant anymore: it's not part of the privacy regulations, it's used by very few people so it's abused by the adtech to target them and it's being phased out from browsers (and even browsers that offer it, don't have it on by default). and tracking it was supposed to prevent, we don't do in the first place

**Reply by mirdaki** [reply]
I respect your decision. Thank you for the response!
